### DISCUSSION TITLE
New projects cannot be deployed

### DISCUSSION THREAD

**Discussion by DamonAsghari** [ConsenSys-archive/truffle#6046] [General Q&A]
Hi,
I have created a new test project on vscode using Truffle, but the contract can not be deployed, neither on the local Ganache blockchain nor on live testnets,
after compiling the message "network up to date" shows up in terminal and that's it.
![Screenshot (349)](https://github.com/trufflesuite/truffle/assets/108937977/2744a496-bb68-4fd8-bed2-0e2ae4042af8)

**Comment by FlaviuWeb3** [comment]
Hi @MiloWinchester could you please share with me a repository link so I can test the above configuration?

**Reply by DamonAsghari** [reply]
this is the repository link https://github.com/MiloWinchester/UnitTesting.js
configuration is as same as my Ganache setting, you can test for yourself.

**Comment by dqwira** [comment]
I noticed something different in the port number you're using. Are you purposely using 8595? Maybe you can try changing the default port to 8545 for g-cli or 7545 for g-gui
